src/openapi.json

```diff
@@ -3,7 +3,11 @@
 	"info": {
 		"title": "Philanthropy Data Commons API",
 		"description": "An API for a common data platform to make the process of submitting data requests to funders less burdensome for organizations seeking grants.",
-		"version": "0.11.0"
+		"version": "0.11.0",
+		"license": {
+			"name": "GNU Affero General Public License v3.0 only",
+			"url": "https://spdx.org/licenses/AGPL-3.0-only.html"
```

Comment:
In place of this I tried `"identifier": "AGPL-3.0-only"` and it seemed to have the same effect locally of putting up this URL. Regardless, OK either way.

Reply:
Oh interesting -- side note, the lack of support for `identifier` seems to just be in our linter tool.  We may eventually find that the swagger linter is out of date and unmaintained, and we need to either remove it or replace it...

Reply:
Ah, I didn't process that detail that it was the validator. Thanks!